src/rules/functionality.ts

```diff
@@ -51,6 +59,10 @@ export default {
   'no-string-throw': true,
   // Disallows falling through case statements.
   'no-switch-case-fall-through': true,
+   // Warns when a method is used as outside of a method call.
+  'no-unbound-method': false,
+   // Warns when using an expression of type ‘any’ in an unsafe way. Type casts and tests are allowed. Expressions that work on all values (such as ‘”” + x’) are allowed.
+  'no-unsafe-any': true,
```

Comment:
Lots of errors. So many that I can't scroll though them all. Probably in the couple hundreds.